File: FitnessWeb.API/ViewModels/FitnessProgramViewModel.cs
namespace FitnessWeb.API.ViewModels
{
    public class FitnessProgramViewModel
    {
        public int Id { get; set; }
        public FitnessTypeViewModel Type { get; set; }
        public List<TrainingViewModel> Trainings { get; set; }
        public List<FitnessTipViewModel> FitnessTips { get; set; }
    }
}

File: FitnessWeb.Frontend/AngularApp/src/viewModels/fitnessProgramViewModel.ts
import { FitnessTipViewModel } from "./fitnessTipViewModel";
import { FitnessTypeViewModel } from "./fitnessTypeViewModel";
import { TrainingViewModel } from "./trainingViewModel";

export class FitnessProgramViewModel {
    id: number;
    type: FitnessTypeViewModel;
    trainings: TrainingViewModel[];
    fitnessTips: FitnessTipViewModel[];

    constructor(id: number, type: FitnessTypeViewModel, trainings: TrainingViewModel[], fitnessTips: FitnessTipViewModel[]) {
        this.id = id;
        this.type = type;
        this.trainings = trainings;
        this.fitnessTips = fitnessTips;
    }
}